### DISCUSSION TITLE
Turnkey Nextclouud LXC

### DISCUSSION THREAD

**Discussion by Gangstrocity** [tteck/Proxmox#1784] [Help Needed] [answered]
When I run the script for the Turnkey Nextcloud LXC it errors out. Any ideas?

[INFO] Using 'local' for template storage.
[INFO] Using 'local-lvm' for container storage.
Updating LXC template list...
Creating LXC container...
Starting LXC Container...
RTNETLINK answers: Network is unreachable
[ERROR] 2@171 Unknown failure occured.
[ERROR] 2@171 Unknown failure occured.

**Comment by tteck** [comment]
In which country are you located?

**Reply by Gangstrocity** [reply]
United States.

**Reply by tteck** [reply]
The error message you're seeing, "RTNETLINK answers: Network is unreachable," typically indicates a problem with network connectivity on a Linux system.

**Reply by tteck** [reply]
This error message is often associated with the `ip` or `route` commands when attempting to manipulate network routes.
I'm exploring an alternative method to obtain the IP address using `pct exec`, with the expectation that it will function for everyone.

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/1742#discussioncomment-6874470

**Reply by Gangstrocity** [reply]
The script doesn't behave like the rest of your scripts. It doesn't set and IP it only asks for storage information then starts creating the container. Once it fails the container is deleted.

**Reply by tteck** [reply]
My intention is to maintain the TurnKey scripts in their simplest form, contained within a single file, and with minimal options, if any.

**Reply by Gangstrocity** [reply]
This ended up working for me. I deleted that one section. It gives an error, but the LXC is created.

**Comment by tteck** [comment]
Are you willing to test new code to see if it works for you? Let me know which TurnKey script you'd like to create.

**Reply by tteck** [reply]
I just pushed this [commit](https://github.com/tteck/Proxmox/commit/3b1560fa63a85ba121995f0bbfe98fe455d2d144) to use an alternative method for acquiring the IP address in the TurnKey scripts.

**Please let me know if this worked.**

**Reply by Gangstrocity** [reply]
I tried the new one and had the same issue. It fails and there is no container created.

**Reply by tteck** [reply]
The same one (could be cached) or a different script?

**Reply by tteck** [reply]
I just pushed this [commit](https://github.com/tteck/Proxmox/commit/c10678e251185c59d073f240474afaa7161c03a8) to disable the 'exit on error' behavior while checking for an IP.

**Reply by Gangstrocity** [reply]
I tried the new commit and it still wasn't working correctly. The best success i had was copying the full script and removing the part you mentioned in the other comment. I went ahead and marked that as the best answer just to move this along. I appreciate your help with this, but in general your scripts are amazing and have saved me tons of time. Much appreciated.
